### PR TITLE
near final SRM 0.1.0 version

### PR DESCRIPTION
**Pull request by adconk**
Hey @jonadata13 - here's the latest code for review in anticipation of ServeRmore 0.1.0.  This request is to merge my personal branch into the main pre-release branch, which also was forked from your branch a number of days back.  Would you be willing to destroy/remove your other individual branches?

You can find updated instructions in both the main README and the layers/README, and the story is here:
https://app.clubhouse.io/origent/story/15282/peer-review-test-suite-validation

Please let me know if you have any questions with this build and deployment process versus the previous one from a few weeks ago.

### CODE REVIEW

**Comment by adconk** [comment]
Hi @jonadata13 - I've merged your PR into this one.  Also, I've made another attempt as simplifying the instructions by merging both READMEs.  Let me know what you think!

File: README.md [hunk 1]
```diff
@@ -50,25 +67,36 @@ For deploying a new Lambda function only (i.e. not including a Lambda _layer_),
...
+Additional Settings for Building the Layer:
+
+* `ami`: The Amazon Machine Image ID. The specific one listed above is required, as it uses Amazon Linux 2 operating system with the Docker agent pre-installed.  Our scripts will pull from DockerHub.
+* `default_security_group`: When creating an EC2 virtual machine instance, a security group is created automatically.  We recommend creating your own, or grabbing an existing security group ID and using that as your default here.  A security group is similar to a firewall, but is wrapped around a group of instances. 
+* `ssh_security_group`: In order for the scripts to work, SSH must be enabled and reachable with the new Virtual Machine the build script creates. We recommend creating a new security group and allowing SSH port 22 inside the security group, and recording the ID here.
+* `subnet`: When creating an EC2 virtual machine instance, it is added to a subnet and provided an IP address.  The subnets list can be found in the EC2 console.  Add the ID to one of them here.
+* `instance_type`: The type detrmines cost and capability of the virtual machine.  The type provided has been tested, but many others could potentially work.
```
Comment (jonadata13) [inline README.md:80]: Spelling error: _determines_

File: README.md [hunk 2]
```diff
@@ -78,35 +106,66 @@ srm lambda destroy
...
+If you're curious what shared libraries outside of the installed R folder is being used in the build environment, you can run the following to get a list:
+```
+ldd /usr/lib64/R/bin/exec/R
+```
 
-## Changelog
+Additional log entries from the runtime layer can be placed in the Lambda Service by adding print commands to the following file and function.  Also, example print statements are provided:
 
-v0.1.0 - Revamped Lambda Service Layers and removed outdated deployment steps
+`FILENAME`: ServeRmore/layer/r-runtime/build/layers/r-runtime/build/layer/R/library/base/R/Rprofile
```
Comment (jonadata13) [inline README.md:145]: Can you add a bullet point for `FILENAME` and `METHOD`?

**Review by jonadata13 — CHANGES_REQUESTED** [review]
@adconk - The updates are looking awesome! Thank you for adding more information regarding the YAML file parameters. I just have two minor requests (see below).

**Comment by adconk** [comment]
Hey @jonadata13 - all set.  I rewrote the section on debugging.  What else ya got?

File: README.md [hunk 3]
```diff
@@ -119,31 +119,41 @@ srm terminate
...
 
-If you're curious what shared libraries outside of the installed R folder is being used in the build environment, you can run the following to get a list:
+There's a way to see which shared libraries are being used in the build environment by running  the following command to get a list:
 ```
-ldd /usr/lib64/R/bin/exec/R
+ldd /opt/R/bin/exec/R
 ```
 
-Additional log entries from the runtime layer can be placed in the Lambda Service by adding print commands to the following file and function.  Also, example print statements are provided:
+There's also a way to introduce print log statements int the Lambda R Runtime layer that will add log entries into AWS CloudWatch from AWS Lambda.  Once inside the Docker container, change directories and view the following file:
```
Comment (jonadata13) [inline README.md:147]: Spelling error: _in_

File: README.md [hunk 4]
```diff
@@ -119,31 +119,41 @@ srm terminate
...
+* aws.s3 - Used to interact with AWS S3 storage buckets.
+* logging - Used to help create well formed log streams.
+* yaml - Used to set configuration settings in a standardized way.
 
 **Note:**: The build and compilation process uses a Docker image called [docker-lambda](https://github.com/lambci/docker-lambda).
 
 ### Base R Runtime Layer Debugging
 
-If there are challenges with the layer build, the following command can be run to interact with the environment inside the docker service used to build the layer. Once inside the Terminal of the Docker container, the individual commands listed in **Dockerfile** can be run. Before running, make sure you have already run `srm deploy` at least once:
+If there are challenges with the layer build, there are ways to enter into an interactive mode. First, make sure that you've already run deploy once before without terminating. Next, check the status to ensure a VM is running. Finally login to the VM itself and then the Docker container through the following commands:
```
Comment (jonadata13) [inline README.md:133]: Comma after 'Finally'

**Review by jonadata13 — CHANGES_REQUESTED** [review]
_Two_ more minor requests!

**Comment by adconk** [comment]
Done! @jonadata13 - got anything else for me?

**Comment by jonadata13** [comment]
@adconk Everything looks good! 🎉

**Pull request merged**